test/system/030-run.bats

```diff
@@ -472,6 +472,14 @@ json-file | f
     is "$output" "$expect" "podman run with --tz=local, matches host"
 }
 
+@test "podman run --tz with zoneinfo" {
+    # First make sure that zoneinfo is actually in the image otherwise the test is pointless
+    run_podman run --rm $SYSTEMD_IMAGE ls /usr/share/zoneinfo
+
+    run_podman run --rm --tz Europe/Berlin $SYSTEMD_IMAGE readlink /etc/localtime
+    assert "$output" == "../usr/share/zoneinfo/Europe/Berlin" "localtime is linked correctly"
```

Comment:
I think you're going to need to add `run_podman rmi $SYSTEMD_IMAGE` here. (And maybe in teardown, but I'm less worried about that). I tried, when creating the systemd image, to fix tests so they would accept it in the image list, but obviously I didn't catch everything.

Reply:
Let's see if #18762 fixes these problems.

Reply:
Oh my, and I was going crazy over here because they pass when I ran them individually. I applied Tom nits and cherry-picked your commit to fix the tests.